.travis.yml

```diff
@@ -4,7 +4,7 @@ language: php
 
 branches:
   except:
-    - /^release-.*$/
+    - /^release-\d+\.\d+\.\d+.*$/
     - /^ghgfk-.*$/
```

Comment:
In theory, this is just a forked repo. All forked repos are not actually copies...